include/vcpkg/portlint.h

```diff
@@ -0,0 +1,53 @@
+#pragma once
+
+#include <vcpkg/base/fwd/files.h>
+#include <vcpkg/base/fwd/stringview.h>
+
+#include <vcpkg/fwd/packagespec.h>
+#include <vcpkg/fwd/triplet.h>
+#include <vcpkg/fwd/vcpkgpaths.h>
+
+#include <vcpkg/sourceparagraph.h>
+
+#include <set>
+
+namespace vcpkg::Lint
+{
+    enum class Status : int
+    {
+        Ok = 0b0,
+        Problem = 0b01,
+        Fixed = 0b10,
+        PartiallyFixed = 0b11
+    };
```

Comment:
Why didn't you just write 0, 1, 2 and 3?
If you really want to safe some space, you could use `std::byte`.

Reply:
Because I want to write 
```c++
Status status = Status::Ok;
if (... true) status |= Status::Problem;
if (... true) status |= Status::Fixed;
// status is now Status::PartiallyFixed
```

Reply:
smart!